framework/src/systems/AuxiliarySystem.C

```diff
@@ -52,13 +52,7 @@ AuxiliarySystem::AuxiliarySystem(FEProblemBase & subproblem, const std::string &
     _elemental_array_aux_storage(_app.getExecuteOnEnum())
 {
   _nodal_vars.resize(libMesh::n_threads());
-  _nodal_std_vars.resize(libMesh::n_threads());
-  _nodal_vec_vars.resize(libMesh::n_threads());
-  _nodal_array_vars.resize(libMesh::n_threads());
   _elem_vars.resize(libMesh::n_threads());
-  _elem_std_vars.resize(libMesh::n_threads());
-  _elem_vec_vars.resize(libMesh::n_threads());
-  _elem_array_vars.resize(libMesh::n_threads());
```

Comment:
Nice consolidation.

Reply:
But honestly, I don't fully understand why they were needed before. They seemed to be holding the same variables.

Reply:
They were used in the unnecessary calls to things like `prepareAux` and also for things like `var->insert` ... but we don't need the former and the latter is better accomplished by calling `kernel->variable().insert`. Why is it better? Because the variable may be in the displaced aux system and not the undisplaced one

Reply:
Okay, circling back to your other comment on `aux_sys` acting working only for undisplaced meshes.